schemas/panels/time-series/time-series.cue

```diff
@@ -26,6 +26,7 @@ import (
 	area_opacity?:  number & >=0 & <=1
 	show_points?:   "Auto" | "Always"
 	point_radius?:  number & >=0 & <=6
+	stack?:         "None" | "Normal" // TODO: add "Percent" option support
```

Comment:
`"None"` isn't a bit redundant since `stack` is an optional field ?

Reply:
Yeah agreed I'll fix this, planning to leave it as an option in the dropdown for visual editing, but selecting 'None' will reset `stack` so it's undefined.

Reply:
This is updated now